src/internal/connector/onedrive/url_cache.go

```diff
@@ -14,11 +14,20 @@ import (
 	"github.com/alcionai/corso/src/pkg/services/m365/api"
 )
 
+type urlCacher interface {
```

Comment:
This naming convention doesn't follow golang standards for interface naming.  Interfaces should be named after their behavior.  `getItemPropertyer`, would be more appropriate in this case.  That naming convention helps to ensure individual interfaces stay small, and can be composed into more complex aggregates as needed.  Starting off with something like `nameOfStructer` inevitably leads to bloated interfaces that duplicate the surface area of the struct, which is a smell.